plugins/image2/plugin.js

```diff
@@ -1697,3 +1697,13 @@ CKEDITOR.config.image2_captionedClass = 'image';
  * @cfg {String[]} [image2_alignClasses=null]
  * @member CKEDITOR.config
  */
+
+/**
+ * Determines whether alternative text is required.
+ *
+ *		config.image2_altRequired = false;
+ *
+ * @since 4.5
```

Comment:
Please target it for 4.6.0 instead (it's important to use 3 digs, as we're using semver).